dio/test/utils.dart

```diff
@@ -74,6 +74,17 @@ Future<void> startServer() async {
         return;
       }
 
+      if (path == '/multi-value-header') {
+        response.headers.contentType = ContentType('application', 'json');
+        response.headers.add("x-multi-value-request-header-echo", request.headers.value("x-multi-value-request-header").toString());
```

Comment:
Please, use single quotes here like this:

`        response.headers.add('x-multi-value-request-header-echo', request.headers.value('x-multi-value-request-header').toString());
`